ui/web/static/js/charts.js

```diff
@@ -1,63 +1,17 @@
 // Copyright (c) Abstract Machines
 // SPDX-License-Identifier: Apache-2.0
 
-function createChart(chartData, config) {
-  switch (chartData.Type) {
-    case "alarmCount":
-      return createAlarmCountCard(chartData, config);
-    case "alarmsTable":
-      return createAlarmsTable(chartData, config);
-    case "areaLineChart":
-      return createAreaLineChart(chartData, config);
-    case "timeSeriesBarChart":
-      return createTimeSeriesBarChart(chartData, config);
-    case "donutChart":
-      return createDonutChart(chartData, config);
-    case "doubleBarChart":
-      return createDoubleBarChart(chartData, config);
-    case "dynamicDataChart":
-      return createDynamicDataChart(chartData, config);
-    case "entitiesTable":
-      return createEntitiesTable(chartData, config);
-    case "entityCount":
-      return createEntityCount(chartData, config);
-    case "gaugeChart":
-      return createGaugeChart(chartData, config);
-    case "horizontalBarChart":
-      return createHorizontalBarChart(chartData, config);
-    case "labelCard":
-      return createLabelCard(chartData, config);
-    case "lineChart":
-      return createTimeSeriesLineChart(chartData, config);
-    case "multiBarChart":
-      return createMultiBarChart(chartData, config);
-    case "multiGaugeChart":
-      return createMultiGaugeChart(chartData, config);
-    case "multipleLineChart":
-      return createMultipleLineChart(chartData, config);
-    case "pieChart":
-      return createPieChart(chartData, config);
-    case "progressBar":
-      return createProgressBar(chartData, config);
-    case "sharedDatasetChart":
-      return createSharedDatasetChart(chartData, config);
-    case "speedGaugeChart":
-      return createSpeedGaugeChart(chartData, config);
-    case "stackedLineChart":
-      return createStackedLineChart(chartData, config);
-    case "stepChart":
-      return createStepChart(chartData, config);
-    case "tempGaugeChart":
-      return createTempGaugeChart(chartData, config);
-    case "valueCard":
-      return createValueCard(chartData, config);
-    default:
-      return null;
-  }
-}
-
-function createAlarmCountCard(chartData, config) {
-  config.Content = `
+class AlarmCount {
```

Comment:
```suggestion
class AlarmCount {
  constructor(chartData, widgetID) {
    this.chartData = chartData;
    this.ID = widgetID;
    this.Style = {
      width: "400px",
      height: "175px",
    };
    this.Content = this.#generateContent();
  }

  #generateContent() {
    return `
      <div class="item-content" id="${this.ID}" style="width: ${this.Style.width}; height: ${this.Style.height};">
        <div class="card widgetcard">
          <div class="card-header text-center">
            <h5 class="card-title">Alarm Count</h5>
          </div>
          <div class="card-body text-center">
            <p class="card-text value">35</p>
          </div>
          <div class="card-footer text-center w-100 bg-warning">
            <p class="card-text">Warning Level</p>
          </div>
        </div>
      </div>
    `;
  }
}
```
Do this for the rest of the bootstrap widgets